site/build/reference/render.js

```diff
@@ -148,18 +151,27 @@ function renderProperties(language, docs, templates) {
     var body = renderText(docs.properties[key], templates).content;
     var source = docs.source.properties[key];
     var id = escapeKey(property.key);
+    var template = templates.entry;
 
     if (uniqueIDs[id]) {
-      throw new Error('Duplicate html id for key: ' + property.key + ' and ' +
+      throw new Error('Duplicate html id for key: ' + property.key + 'and' +
                       uniqueIDs[id]);
     }
     uniqueIDs[id] = property.key;
 
-    rendered[key] =
-      renderEntry(id, heading, body, property, source, templates.entry);
+    if (references.hasOwnProperty(key)) {
```

Comment:
I could declare `var reference = references[key]` higher up, and then assert in the `property` and `cast` clause that `reference` is defined. (But I don't know if this is the right file to be asserting...)